docs/src/eww.md

```diff
@@ -37,6 +37,18 @@ The following list of package names should work for arch linux:
 
 </details>
 
+<details>
+<summary>VoidLinux</summary>
+
+- cairo-devel 
+- pango-devel 
+- glib-devel 
+- atk-devel 
+- gtk+3-devel 
+- gdk-pixbuf-devel
```

Comment:
Only the packages `gtk+3-devel` and `glib-devel` are required. The rest are dependencies of the aforementioned packages

Reply:
still it might give a hand for newbies if they face any issues

Reply:
> still it might give a hand for newbies if they face any issues

They probably won't find any